pylint/pyreverse/diagrams.py

```diff
@@ -121,16 +121,18 @@ def get_relationship(
     def get_attrs(self, node: nodes.ClassDef) -> list[str]:
         """Return visible attributes, possibly with class name."""
         attrs = []
-        properties = [
-            (n, m)
+        properties = {
+            n: m
             for n, m in node.items()
             if isinstance(m, nodes.FunctionDef) and decorated_with_property(m)
-        ]
-        for node_name, associated_nodes in (
-            list(node.instance_attrs_type.items())
-            + list(node.locals_type.items())
-            + properties
+        }
+        for n, m in list(node.locals_type.items()) + list(
+            node.instance_attrs_type.items()
         ):
+            if n not in properties:
+                properties[n] = m
```

Comment:
Let's use explicit name instead, I don't know what n/m is and I took 30s to try to come up with a proper suggestion: definitely a read blocker 😄

Reply:
I'm not totally sure what `m` is, so I went with `name` and `attr`.